### PR TITLE
Add support package page

### PR DESCRIPTION
**Pull request by apburnes**
## Changes proposed in this pull request:
- Add support package page
- Add support package data with info about package description, price, and hours
- Add link under the pricing tab on the header

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/support-package-page/support-packages/)

## Security Considerations
None

### CODE REVIEW

**Review by eddietejeda — CHANGES_REQUESTED** [review]
For FAQ, we should make the language more accessible. Here are some changes I recommend, but please check with the team first.



What happens to the money that is not used within the fiscal year or period of performance?
Any money that is not used will be returned to the agency through the "deobligation" process.
    
    
If an agency does not use all the allocated support hours, would they be able to reduce paid support hours for future years?
Yes, we see support hours as a good way for an agency to get started with the platform. Once your team is comfortable with the platform, you should reduce the number of paid support hours.


Is there a way to roll over unused hours to the next fiscal year?
Yes, it's possible. But that depends on the agency's authority and funds. For example, some money can cross fiscal year ("no-year money") while other funds must be used within a set time frame ("3-year money").


What are the limits of the paid support packages?
We will not do direct development on the agency's application. We will only help, analyze, and assist.

File: _pages/support-packages.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+         <div class="tablet:grid-col-9 usa-prose">
+            <h2>FAQ</h2>
+         </div>
+      </div>
+      <div class="grid-row grid-gap">
+         <div class="tablet:grid-col-9 usa-prose">
+            <ol class="column-single">
+               <li class="font-serif-lg">
+                  <h3>Would unspent monies be deobligated at the end of the FY/PoP?</h3>
+                  <p class="font-body-md">Yes, unspent monies need to be deobligated.</p>
```
Comment (ccostino) [inline _pages/support-packages.md]: For some reason there is an extra blank line being rendered above this answer; the other answers don't have the extra space.  I'm not sure why, though!

File: _pages/support-packages.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+               <li class="font-serif-lg">
+                  <h3>Is there a way to roll over unused hours to the next FY?</h3>
+                  <p class="font-body-md">
+                    This depends on the agency and what type of money (“no-year”, “3-year”) the agency is using.
+                  </p>
+               </li>
+               <li class="font-serif-lg">
+                  <h3>What boundaries exist that we won't cross?</h3>
+                  <p class="font-body-md">
+                    We will not do direct development on the agency's application. We will only help, analyze, and assist.
```
Comment (ccostino) [inline _pages/support-packages.md:95]: Is this fair to say instead to help clarify this more?

```suggestion
                    We will not perform direct development on the agency's application. We will only help, analyze, and assist with improving application performance and stability on the platform.
```

**Review by ccostino — COMMENTED** [review]
Thanks, @apburnes!  I noticed one formatting thing and made a suggestion on the last Q&A, if it helps at all.

**Review by eddietejeda — APPROVED** [review]

**Pull request merged**